### PR TITLE
Dialogue: clean format when pasting speaker label

### PR DESCRIPTION
**Pull request by retrofox**
This PR fixes an issue when pasting text with format in the speaker input.

#### Changes proposed in this Pull Request:

* Dialogue: clean format when pasting speaker label

#### Testing instructions:

* Start to edit a conversation/dialogue composition
* Copy a text with format style, for instance: **_RetroFox_**
* Confirm the text is pasted applying the format, but generating a dirty text.
* Confirm with these changes the format style is removed 

**before** | **after**
-------|---------
image | image

#### Proposed changelog entry for your changes:

*

### CODE REVIEW

File: projects/plugins/jetpack/extensions/blocks/conversation/utils.js [hunk 1]
```diff
@@ -8,3 +13,7 @@ export function getParticipantByLabel ( participants, participantLabel ) {
 	const part = participants.filter( ( { label } ) => ( label?.toLowerCase() === participantLabel?.toLowerCase() ) );
 	return part?.length ? part[ 0 ] : null;
 }
+
+export function cleanFormatStyle( html ) {
+	return getTextContent( create( { html } ) )?.trim();
```
Comment (allilevine) [inline projects/plugins/jetpack/extensions/blocks/conversation/utils.js:18]: It seems strange that we need to convert the string into a `RichText` value to remove the HTML tags and turn it back into a string. But it's working 😂

**Review by allilevine — APPROVED** [review]
Fixes the issue and removes the formatting and tags ✅ 
Without the changes | With the changes
--------------------- | -----------------
Screen Shot 2021-02-09 at 5 25 01 PM | Screen Shot 2021-02-09 at 5 28 11 PM

**Review by jeherve — APPROVED** [review]

**Comment by retrofox** [comment]
r220807-wpcom

**Comment by jeherve** [comment]
Cherry-picked to `jetpack/branch-9.4` in 8705df1a701a77711e537e906ab062996de80d17

**Pull request merged**